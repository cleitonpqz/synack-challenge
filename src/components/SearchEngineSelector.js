import React from 'react';

const SearchEngineSelector = () => {
  const onEngineChange = (event) => {
    console.log(event.target.value);
  };
  return (
    <div>
      <select
        name="search-engine"
        id="search-engine-selector"
        onChange={onEngineChange}
      >
        <option value="">--Please choose an option--</option>
        <option value="Bing">Bing</option>
        <option value="Google">Google</option>
        <option value="Both">Both</option>
      </select>
    </div>
  );
};
export default SearchEngineSelector;
