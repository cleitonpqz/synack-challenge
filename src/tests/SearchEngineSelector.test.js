import React from 'react';
import { render } from './test-utils';
import SearchEngineSelector from '../components/SearchEngineSelector';

it('should render SearchEngineSelector correctly', () => {
  const component = render(<SearchEngineSelector />);
  expect(component).toMatchSnapshot();
});
