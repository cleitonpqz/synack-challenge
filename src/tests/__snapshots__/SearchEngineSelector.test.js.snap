// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render SearchEngineSelector correctly 1`] = `
<div>
  <select
    id="search-engine-selector"
    name="search-engine"
    onChange={[Function]}
  >
    <option
      value=""
    >
      --Please choose an option--
    </option>
    <option
      value="Bing"
    >
      Bing
    </option>
    <option
      value="Google"
    >
      Google
    </option>
    <option
      value="Both"
    >
      Both
    </option>
  </select>
</div>
`;
